consensus/ConsensusReporter.py

```diff
@@ -157,21 +158,21 @@ def _write_opposites_line(self, variant):
         """
         classifications = {lab: variant[lab] for lab in self.labs if type(variant[lab]) == str}
         self.report.write(
-            '"{}","{}","{}","{}","{}","{}","{}'.format(variant.chromosome, variant.start, variant.ref, variant.alt,
-                                                       variant.gene, variant.transcript, variant.c_dna))
+            '{}\t{}\t{}\t{}\t{}\t{}\t{}'.format(variant.chromosome, variant.start, variant.ref, variant.alt,
+                                                variant.gene, variant.transcript, variant.c_dna))
         for lab in self.labs:
             if lab in classifications:
-                self.report.write('","' + classifications[lab])
+                self.report.write('\t' + classifications[lab])
             else:
-                self.report.write('","')
-        self.report.write('"\n')
+                self.report.write('\t')
+        self.report.write(os.linesep)
```

Comment:
I prefer the previous solution where the output is equal regardless of the operating system that it is run on.